app/src/main/java/com/example/happymeals/database/FireAuth.java

```diff
@@ -0,0 +1,121 @@
+package com.example.happymeals.database;
+
+
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+
+import com.example.happymeals.OutputListener;
+import com.google.firebase.auth.AuthResult;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.firestore.DocumentReference;
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.android.gms.tasks.Task;
+
+import com.google.firebase.firestore.FirebaseFirestore;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/*
+
+ */
+
+
+public class FireAuth {
```

Comment:
Don't need shorthand class names, should be FirebaseAuthenticationHandler, or just AuthenticationHandler/Manager/etc.